pkg/scheduler/server/api.go

```diff
@@ -44,19 +42,21 @@ func (s *Server) ScheduleJob(ctx context.Context, req *schedulerv1pb.ScheduleJob
 	if err != nil {
 		return nil, fmt.Errorf("error parsing due time: %w", err)
 	}
-	ttl, err := parseTTL(req.GetJob().GetTtl())
-	if err != nil {
-		return nil, fmt.Errorf("error parsing TTL: %w", err)
-	}
+	// TODO: ONLY add expiration if ttl is parsed
+
+	//ttl, err := parseTTL(req.GetJob().GetTtl())
```

Comment:
I am adding a PR for this.

Reply:
FYI - I commented this out bc by upgrading to the latest cron lib, this now needs some work here that Artur has locally to resolve this